.lycheeignore

```diff
@@ -27,6 +27,10 @@ https?://twitter\.com(?:/.*$)?$
 # An example OpenSAFELY project that is not intended to exist.
 https?://github\.com/opensafely/example-research-repository(?:/.*)?$
 
+# digital.nhs.uk links used to work but are now failing with a 403
+# Network error: Forbidden
+https?://digital.nhs.uk(?:/.*$)?$
```

Comment:
nit: I'd perhaps move this into the section above:

> `# Block either the GitHub Actions runner or the lychee client.`

rather than list it as a special case.

In my experience of updating `.lycheeignore`, this behaviour happens with quite a lot of websites. The checks initially pass, but later get blocked entirely.

(Whether sites are specifically targeting blocking of lychee or GitHub Actions IP address ranges, or just all bots as there are probably more aggressively scraping bots for AI content these days, I don't know.)